modules/directives/src/main/scala/scala/build/preprocessing/directives/Entrypoints.scala

```diff
@@ -0,0 +1,27 @@
+package scala.build.preprocessing.directives
+
+import scala.build.directives.*
+import scala.build.errors.BuildException
+import scala.build.options.{BuildOptions, PostBuildOptions}
+import scala.cli.commands.SpecificationLevel
+
+@DirectiveExamples("//> using entrypoints")
+@DirectiveUsage("//> using entrypoints", "`//> using entrypoints`")
+@DirectiveDescription("Enable entrypoint annotation support")
+@DirectiveLevel(SpecificationLevel.EXPERIMENTAL)
+final case class Entrypoints(
```

Comment:
Sorry for the drive-by comment, but couldn't this be added just by using `//> using lib "com.github.alexarchambault::case-app-entrypoint-annotation:0.1.0-SNAPSHOT"` ? Is a separate directive needed?

Reply:
Having the using directive should make the completion stuff try to actually compile the project, and provide completions for the entrypoints' options. Ideally, it should also import the annotation itself.